src/resources/views/base/inc/alerts.blade.php

```diff
@@ -1,21 +1,43 @@
 {{-- Bootstrap Notifications using Prologue Alerts & PNotify JS --}}
 <script type="text/javascript">
-  Noty.overrideDefaults({
-    layout   : 'topRight',
-    theme    : 'backstrap',
-    timeout  : 2500, 
-    closeWith: ['click', 'button'],
-  });
 
-  @foreach (\Alert::getMessages() as $type => $messages)
+// This is intentionaly run after dom loads so this way we can avoid showing duplicate alerts
+// when the user is beeing redirected by persistent table, that happens before this event triggers.
 
-      @foreach ($messages as $message)
+window.addEventListener('DOMContentLoaded', function() {
+    //get the php alerts
+    var $crudAlerts = JSON.parse('{!! json_encode( \Alert::getMessages() , JSON_HEX_APOS) !!}');
```

Comment:
I'd call this `backpackAlerts` instead of `crudAlerts`, since they're not specific to CRUDs, they can also be triggered in pages that don't use CRUD.

Also I wonder if we can't use the Blade helper here `@json(\Alert::getMessages())` - is the result different from `{!! json_encode( \Alert::getMessages() , JSON_HEX_APOS) !!}`? Interesting to know.